client/src/pages/Home.tsx

```diff
@@ -127,25 +127,42 @@ function Home(this: any) {
                         </svg>
                     </button>
 
-                    <ul className="rounded absolute hidden text-gray-700 pt-1 group-hover:block w-56" style={{ zIndex: 3 }}>
-                        <li className={"bg-gray-200 hover:bg-gray-400 py-4 px-4 cursor-pointer"} onClick={() => handleFontClick("open-sans")}>
+                    <ul
+                        className="rounded absolute hidden text-gray-700 pt-1 group-hover:block w-56"
+                        style={{ zIndex: 3 }}
+                    >
+                        <li
+                            className={"bg-gray-200 hover:bg-gray-400 py-4 px-4 cursor-pointer"}
+                            onClick={() => handleFontClick("open-sans")}
+                        >
                             Open Sans
                         </li>
-                        <li className="bg-gray-200 hover:bg-gray-400 py-4 px-4 cursor-pointer" onClick={() => handleFontClick("roboto")}>
+                        <li
+                            className="bg-gray-200 hover:bg-gray-400 py-4 px-4 cursor-pointer"
+                            onClick={() => handleFontClick("roboto")}
+                        >
                             Roboto
                         </li>
-                        <li className="bg-gray-200 hover:bg-gray-400 py-4 px-4 cursor-pointer" onClick={() => handleFontClick("serif")}>
-                            Times New Roman
+                        <li
+                            className="bg-gray-200 hover:bg-gray-400 py-4 px-4 cursor-pointer"
+                            onClick={() => handleFontClick("mooli")}
+                        >
+                            Mooli
                         </li>
                     </ul>
                 </div>
             </div>
             <div className="flex justify-center gap-5 p-10">
                 {/* Left text box */}
-                <div className="text-box" style={{ position: "relative" }}>
+                <div
+                    className="text-box"
+                    style={{ position: "relative" }}
```

Comment:
you can use the tailwindcss class "relative" here 👍, this can be applied to wherever "relative" appears